src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml

```diff
@@ -83,7 +87,8 @@
                                    ShowLineNumbers="True"
                                    Background="#272822"
                                    Foreground="#F8F8F2"
-                                   HorizontalScrollBarVisibility="Disabled" />
+                                   HorizontalScrollBarVisibility="Disabled"
+                                   Options="{Binding TextOptions, FallbackValue={StaticResource TextEditOptions}}" />
```

Comment:
@zeusongit If you remove this binding - does everything still work?

Reply:
yes, it does -.-